examples/tests/visual.test.ts

```diff
@@ -0,0 +1,45 @@
+// visual test for github.com
+
+import { shortest } from "@antiwork/shortest";
+
+// GitHub header visual test
+shortest("Visit github.com and verify the global navigation header layout. Check GitHub logo, search bar, navigation items (Pull requests, Issues, Marketplace, Explore), and profile dropdown maintain correct spacing and alignment", {
```

Comment:
Giving AI long format test description. This could be multiple test cases to check each of these condition separately as well.